### ISSUE TITLE
Textarea does not mark as dirty when its value autofilled by browser (WebKit, Chromium 43)

### ISSUE THREAD

**Issue by atstek** [google/material-design-lite#1152]
As in the cases of issues [#903](https://github.com/google/material-design-lite/issues/903) and [#959](https://github.com/google/material-design-lite/issues/959) textfield component does not notified about value change.

When page is just loaded:
![Screenshot 1](http://i.imgur.com/dU1KCB3.png)

And when you open suggestions dropdown menu:
![Screenshot 2](https://lh6.googleusercontent.com/xLdaz8Po3s1orHY93QAn_n74Y69Zs42XQlbNB5PLEg-mChNxGRmix-_ByuVPJPRJT6zxdoTGGmB6nHM=w1351-h693)

Actually Chromium does not change the value of input at that moment. It just insert possible values as many autocomplete featueres are working. While 'change' event fired only after actually selecting the value.

Demo: [http://codepen.io/anon/pen/YXOVOW](http://codepen.io/anon/pen/YXOVOW)
You can fill the form with any values and reload the page.

**Comment by Garbee** [comment]
Duplicate of the issues mentioned in the report.

**Comment by atstek** [comment]
Yes, I saw the solutions for those issues. Anyway, I could not kill this two birds with one stone. Although the causes of this issues are the same, initiators are different. In one case it is JS code, in other the browser itself. Seems the general solution is scanning all inputs to detect changes. But it does not completely solve the problem in the case with a browser, even though it is bad idea in itself.
I have found 
I have made a temporary workaround (https://gist.github.com/atstek/c3e81724f147010d7de0) that scan inputs of certain form to detect \:autofill, \:-webkit-autofill, \:-moz-autofill pseudo classes only while any of these inputs has focus and notify me. It may be used like so
```css
.visibility--hidden {
    visibility: hidden;
}
```
```javascript
var autofillMonitor = AutofillMonitor.scan(form, {
    onAutofill: function(inputs) {
        $(inputs).siblings('label').addClass('visibility--hidden');
    },
    onClear: function(inputs) {
        $(inputs).siblings('label').removeClass('visibility--hidden');
    }
});
...
autofillMonitor.stop();
...
autofillMonitor.start();
```
I started work with one project using MDL (at my own risk :smile:), so looking forward to solving these issues.
